test/concurrent/ThreadTest.ooc

```diff
@@ -45,15 +45,17 @@ ThreadTest: class extends Fixture {
 				value set(value get() + 1)
 				Thread yield()
 				Time sleepMilli(1)
+				if (Thread currentThread() isCanceled())
```

Comment:
By `global thread tables` I mean that `Thread currentThread()` could return the same `instance` each time, assuming that instances are stored in a `thread Id -> thread instance` global hash table. Do you find my idea as ugly as I do @marcusnaslund ? :)